src/components/CheckoutForm/SelectInput

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
import React from 'react';

Reply:
in new version react you not need that )